recipes/oatpp-swagger/all/conanfile.py

```diff
@@ -43,7 +47,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires(f"oatpp/{self.version}")
+        self.requires(f"oatpp/{self.version}", transitive_headers=True)
```

Comment:
I just saw the errors in the CI. Reverting the previous change and keeping it as it was:

```suggestion
        self.requires(f"oatpp/{self.version}", transitive_headers=True, transitive_libs=True)
```